.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+version: 2.1
+
+orbs:
+  ruby: circleci/ruby@1.7.1
+  node: circleci/node@2
+  browser-tools: circleci/browser-tools@1.2.5
+  
+commands:
+  bundle_cache:
+    steps:
+      - restore_cache:
+          key: gem-cache-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+      - save_cache:
+          key: gem-cache-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+
+jobs:
+  lintering:
+    docker:
+      - image: cimg/ruby:2.7.3-node
+    steps:
+      - checkout
+      - bundle_cache
+      - run: bundle install 
+      - run:
+          name: rubocop
+          command: bundle exec rubocop
+          
+
+  test:
+    docker:
+      - image: cimg/ruby:2.7.3-node
+      - image: circleci/postgres:12-alpine
+        environment:
+          POSTGRES_USER: johnnyGsy
+          POSTGRES_DB: rails_7_bookstore_test
+          POSTGRES_PASSWORD: "password"
+    environment:
+      BUNDLE_JOBS: "3"
+      BUNDLE_RETRY: "3"
+      PGHOST: 127.0.0.1
+      BOOK_STORE_DATABASE_PASSWORD: "password"
+      RAILS_ENV: test
+    steps:
+      - checkout
+      - bundle_cache
+      - run: bundle install 
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
+      - run:
+          name: Install ImageMagick
+          command: sudo apt-get install imagemagick
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
+      - run:
+          name: Database setup
+          command: bundle exec rails db:schema:load --trace
+      - run:
+          name: run specs
+          command: bundle exec rspec
```

Comment:
if you use parallelism you need to split specs to different containers in another case it doesn't matter